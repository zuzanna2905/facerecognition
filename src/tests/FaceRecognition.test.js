import {shallow} from 'enzyme';
import React from 'react';
import FaceRecognition from '../component/FaceRecognition/FaceRecognition';

it('renders FaceRecognition without crashing', () => {
    const initialProps = {
        imageUrl: '',
        box : {
            topRow : 0,
            rightCol : 0,
            bottomRow : 0,
            leftCol: 0
        }
    }
    let wrapper = shallow(<FaceRecognition {...initialProps}/>);
    expect(wrapper).toMatchSnapshot();
});