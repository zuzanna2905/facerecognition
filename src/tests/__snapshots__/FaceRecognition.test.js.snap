// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FaceRecognition without crashing 1`] = `
<div
  className="center ma"
>
  <div
    className="absolute mt2"
  >
    <img
      alt=""
      height="auto"
      id="inputImage"
      src=""
      width="500px"
    />
    <div
      className="bounding-box"
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
</div>
`;
